src/providers/FiltersProvider.tsx

```diff
@@ -70,6 +70,7 @@ const imageFilterSchema = z.object({
   followed: z.boolean().optional(),
   tools: z.number().array().optional(),
   techniques: z.number().array().optional(),
+  baseModels: z.enum(constants.baseModels).or(z.enum(constants.baseModels).array()).optional(),
```

Comment:
Let's just accept array in here
```suggestion
  baseModels: z.enum(constants.baseModels).array().optional(),
```